app/components/Table/TableRowCell.jsx

```diff
@@ -34,13 +34,12 @@ export default class TableRowCell extends React.Component {
 
     return (
       <td className={cellStyle.getClasses()}>
-        {React.Children.map(this.renderChildren(), child =>
-          // If the child is a HTMLElement (type is a string), don't do anything.
-          // Otherwise the child is a React Component; pass the onHover callback.
-          typeof child.type === 'string'
-            ? child
-            : React.cloneElement(child, {onHover})
-        )}
+        {React.Children.map(this.renderChildren(), child => {
+          // Only pass onHover to field components.
+          return child.type.name.startsWith('Field')
```

Comment:
Is this a good idea? 🤔 
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
(I don't have a better one.)

Reply:
I can try and come up with a better one when there's less work :man_shrugging: taking a note